samples/MasterDetail/MasterDetailSample/MasterDetailSample/MasterDetailSample.csproj

```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.5.0.617" />  
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.2" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.495" />  
```

Comment:
Why such a low version? Is anything a new problem in the newest xamarin version?

Reply:
I just consolidated the versions and don't want to increase it for the whole framework without testing it.